.changeset/gorgeous-kings-cough.md

```diff
@@ -0,0 +1,5 @@
+---
+'@commercetools-backend/express': patch
+---
+
+Now that users can request the Merchant Center API to include logged in user's permissions in the exchange JWT included in proxied requests, our _auth_ middleware will check for that information in the token and will make it available in the `request.session` object under a new property named **userPermissions**.
```

Comment:
Nit: we can also add a link to the docs (even though they are not live yet).

Reply:
Updated in this commit: b80cd139edcc473ceff46de82772a70329dcecc8